generic/Tensor.c

```diff
@@ -1319,6 +1319,7 @@ void torch_Tensor_(init)(lua_State *L)
   luaT_setfuncs(L, torch_Tensor_(_), 0);
   lua_pop(L, 1);
   THVector_(vectorDispatchInit)();
+  THTensor_(tensorMathDispatchInit)();
```

Comment:
Also, could you please merge these into a single function? We should have one entry point for initializing the function pointers, or we'll have people forgetting about them.